src/pages/PlayClass.js

```diff
@@ -0,0 +1,16 @@
+import { useParams } from "react-router-dom";
+import PlayClassCard from "../components/PlayClassCard/PlayClassCard";
+
+export default function PlasClass({ data, toggleFavourite }) {
+  const { id } = useParams();
+
+  const classToPlay = data.filter((savedClass) => savedClass.id === id);
+
+  return (
+    <PlayClassCard
+      id={id}
```

Comment:
You might be better off calling it pageId or itemId to not confuse it with an html element id at some point.